solc/jsonCompiler.cpp

```diff
@@ -305,6 +307,11 @@ static string s_outputBuffer;
 
 extern "C"
 {
+extern char const* license()
+{
+	/// TOOD: include the copyright information on the top.
+	return licenseText;
```

Comment:
Actually this might hilight a bug in the other pull request. We should manually zero-terminate the character array (as part of the template). Can you do this together with this pull request?